### DISCUSSION TITLE
Working install guide 14 Nov 2022

### DISCUSSION THREAD

**Discussion by anthonyuk** [automatic-ripping-machine/automatic-ripping-machine#616] [General]
This is what I used this weekend to get a working ARM install.

Install Ubuntu 20.04
During install create user arm password arm (or whatever)
Login With user arm
sudo apt update
sudo apt upgrade

wget https://raw.githubusercontent.com/automatic-ripping-machine/automatic-ripping-machine/v2_devel/scripts/installers/ubuntu-20.04-install.sh
chmod +x ubuntu-20.04-install.sh

sudo ./ubuntu-20.04-install.sh

sudo nano /etc/systemd/system/armui.service
Change ExecStart=python3 /opt/arm/arm/runui.py to ExecStart=/usr/bin/python3 /opt/arm/arm/runui.py

ctrl+o to save, ctrl+x to exit

ln -s /etc/.abcde.conf /etc/arm/config/abcde.conf

Reboot

**Comment by jack-dvd-ripper** [comment]
This helped me so very much. Thank you!
